core-services/prow/02_config/_plugins.yaml

```diff
@@ -3469,6 +3490,33 @@ plugins:
   - wip
   - yuks
   - approve
+  devfile/integration-tests:
+  - assign
+  - blunderbuss
+  - bugzilla
+  - cat
+  - dog
+  - heart
+  - golint
+  - goose
+  - help
+  - hold
+  - jira
+  - label
+  - lgtm
+  - lifecycle
+  - override
+  - pony
+  - retitle
+  - shrug
+  - sigmention
+  - skip
+  - trigger
+  - verify-owners
+  - owners-label
+  - wip
+  - yuks
+  - approve
```

Comment:
So you are enabling merge-related plugins like `lgtm` and `approve` but not automated merges themselves. Is that intentional?

Reply:
Correct. I'd like to keep merge as a manual process for now. Thanks,